public/app/features/variables/editor/reducer.test.ts

```diff
@@ -188,6 +188,7 @@ describe('variableEditorReducer', () => {
         .thenStateShouldEqual({
           ...initialVariableEditorState,
           extended: {
+            // @ts-ignore - temp ignoring this, we'll fix it soon
```

Comment:
Hmm, this looks weird

Reply:
Because `extended` is now explicitly typed more than just `{}`, `someProp` is no longer a valid here. In hindsight I'm not sure why I didn't just fix this test in this PR, but I did [fix it as intended in the next PR](https://github.com/grafana/grafana/pull/44957/files#diff-c48541823dee41f59d8b2790e7ddb908eb8455755f7b379c53e96ae277a4f41eL191).

I'm hoping we could just leave this as is "for now" given I fix it immediately in the next PR?

Reply:
Thanks for the explanation and I've approved the PR anyway 👍